src/oic/utils/keyio.py

```diff
@@ -782,12 +786,12 @@ def dump(self):
     def restore(self, info):
         for issuer, keys in info.items():
             self.issuer_keys[issuer] = [self.keybundle_cls(
-                keys, verify_ssl=self.verify_ssl)]
+                keys, verify_ssl=self.verify_ssl, timeout=self.timeout)]
 
     def copy(self):
         copy_keyjar = KeyJar(verify_ssl=self.verify_ssl)
```

Comment:
The new KeyJar should inherit the timeout too.

Reply:
True.